homeassistant/components/plant.py

```diff
@@ -250,6 +250,11 @@ def _check_max(self, sensor_name, value, params):
             # only use the database if it's configured
             self.hass.async_add_job(self._load_history_from_db)
 
+        for entity in self._sensormap:
```

Comment:
We should call the variable `entity_id`, since that is what it is.

Reply:
Good call. Done.